src/rsz/src/Resizer.cc

```diff
@@ -746,17 +746,48 @@ void Resizer::findBuffers()
   }
 }
 
+LibertyCell* Resizer::selectBufferCell(LibertyCell* user_buffer_cell)
+{
+  // Prefer user-specified buffer cell if provided.
+  if (user_buffer_cell) {
+    return user_buffer_cell;
+  }
+
+  // Otherwise, find the weakest buffer with the lowest drive resistance.
+  findBuffers();  // updates buffer_lowest_drive_
+
+  // No buffer?
+  if (buffer_lowest_drive_ == nullptr) {
+    logger_->error(RSZ, 23, "No buffers found.");
+  }
+
+  return buffer_lowest_drive_;
+}
+
 bool Resizer::isLinkCell(LibertyCell* cell) const
 {
   return network_->findLibertyCell(cell->name()) == cell;
 }
 
 ////////////////////////////////////////////////////////////////
 
-void Resizer::bufferInputs()
+void Resizer::bufferInputs(LibertyCell* buffer_cell, bool verbose)
 {
   init();
-  findBuffers();
+
+  // Use buffer_cell. If it is null, find the buffer w/ lowest drive resistance.
+  LibertyCell* selected_buffer_cell = selectBufferCell(buffer_cell);
+  if (selected_buffer_cell == nullptr) {
```

Comment:
I don't think this can be null (selectBufferCell would error in that case)

Reply:
When Logger::error is called an exception is thrown (https://github.com/The-OpenROAD-Project/OpenROAD/blob/e4f795a39cceb3608a6e755000253af491a69a3f/src/utl/include/utl/Logger.h#L160).  No normal return can happen with a null value.